generate_version_file.py

```diff
@@ -0,0 +1,3 @@
+from setuptools_scm import get_version
+
+get_version(root="./code/", relative_to="__file__", write_to="src/lando/version.py")
```

Comment:
Can we use `settings.BASE_DIR` to derive these values somehow?

Reply:
After moving the logic to be inside the management command, the `root` no longer needs to be specified at all.